docs/ur/README.md

```diff
@@ -0,0 +1,351 @@
+* Try to avoid long, nested sentence constructions. If necessary, break up the original sentence and regroup it syntactically.
+
+```
+
+Example: Fast for Good.
+Wrong: اچھے کے لئے تیز۔
+Correct: تیز، بہتری کے لئے۔
+
+```
+
+* Use wording that is succinct, unambiguous, and free of jargon.
+* Produce a translation that sounds as it if was originally written in your language, i.e. avoid following the original source sentence structure too closely.
+* Always bear in mind who your target audience is (i.e. an experienced computer user, a beginner, or a combination of both groups).
+* Use a consistent style throughout all product components and across a product range, to ensure that all Mozilla products can be linguistically identified as part of a group of products.
+* Use Formal expressions.
+
+```
+
+Example: Your account is safe
+Wrong: تمہارا اکاؤنٹ محفوظ ہے
+Correct: آپ کا اکاؤنٹ محفوظ ہے
+
+```
+
+## Reference Terminology
+
+The following terminology sources should be used as reference in the translation:
+* Urdu [Glossary](glossary.md)
+* Product-specific glossary, to ensure consistency across all product components visit [Transvision](https://transvision.mozfr.org/?recherche=Mozilla&repo=gecko_strings&sourcelocale=en-US&locale=ur&search_type=strings).
+* Previous version product-specific glossary, visit [Transvision](https://transvision.mozfr.org/?recherche=Mozilla&repo=gecko_strings&sourcelocale=en-US&locale=ur&search_type=strings), to ensure consistency between versions.
+* Glossaries of other Mozilla products, to ensure cross-product consistency.
+Microsoft glossary, to ensure adherence to the industry standards. It is your responsibility to make sure that you always have the latest Microsoft glossaries at your disposal. The glossaries can be found at: http://www.microsoft.com/Language
+* Wordpress Glossary, https://translate.wordpress.com/languages/ur/default/glossary
+
+## General Guideline:
+
+* Access Key / Shortcuts should be in english, old ones need to be refactored eg. T =>> T
+
+```
+[Screenshot]
+```
+
+* XML tags and External Arguments should remain same. XML tag =>
+
+```
+
+<img data=>l10n=>name="icon"/> External Argument => {$name}
+
+```
+
+* Numbers should be same as english numbers.
+
+```
+
+eg. 2 will be 2 and not ٢
+Wrong : ١٢٣٤٥٦٧٨٩١٠
+Correct : 12345678910
+
+```
+
+* For urdu use these symbols:
+
+```
+
+.(full stop) => ( ۔ ) - urdu full stop
+
+,(comma) => ( ، ) urdu comma
+
+brackets should be opposite,
+eg ( name ) =>>... ( نام )
+
+```
+
+## Brand & Product Names
```

Comment:
Please refer to list of trademarks referenced here instead of the list below: https://mozilla-l10n.github.io/styleguides/mozilla_general/#brands-copyright-and-trademark

Reply:
So, should we add the link here? Instead of writing them

Reply:
Yes please

Reply:
Done. Please review.